architecture/rust/jack-float.rs

```diff
@@ -150,10 +150,10 @@ fn main() {
         let output0: &mut[f32] = &mut out_a_p;
         let output1: &mut[f32] = &mut out_b_p;
 
-        let inputs = &[input0, input1];
-        let outputs = &mut[output0, output1];
+        let inputs = vec![input0, input1];
+        let mut outputs = vec![output0, output1];
```

Comment:
Why use a vec here? This is probably a heap allocation, even if it's just a small one.

Reply:
it was a leftover change, sorry